src/app/app-interactive-map/components/app-reaction/app-reaction.component.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, OnInit} from '@angular/core';
+import {Reaction, Reactions} from '../../../../types/custom_types';
+
+@Component({
+  selector: 'app-reaction',
+  templateUrl: './app-reaction.component.html',
+  styleUrls: ['./app-reaction.component.scss'],
+})
+export class AppReactionComponent implements OnInit {
+  public itemsSelected: Reactions = {
```

Comment:
To separate the panels, I created this object with the differents panels, and then, when you add something, is added to the panel that you choose.

If you are in added reactions, the reactions will be added to itemsSelected added.

Reply:
I don't get why you need this. Can you connect the store?

Reply:
It was three days ago, now I know how is the store working and I have to change everything haha